files/openmw.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Alexandre Moine <nobrakal@gmail.com> -->
+<application>
+ <id type="desktop">openmw.desktop</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license> GPL-3.0 and MIT and zlib</project_license>
```

Comment:
1) That will be updated in 0.37 release, but still holds true for 0.36.1 release.
2) We use the system library of tinyxml, but the files still exist in our sources. We have to appease the arm-chair lawyers and cover that too.